src/main/java/org/mastodon/mamut/fitting/FitEllipsoidPlugin.java

```diff
@@ -162,30 +171,119 @@ private void updateEnabledActions()
 		fitSelectedVerticesAction.setEnabled( appModel != null );
 	}
 
-	@SuppressWarnings( { "unchecked", "rawtypes" } )
-	private void fitSelectedVertices()
+	void fitSelectedVertices()
 	{
 		// TODO: parameters to select which source to act on
 		final int sourceIndex = 0;
 
-//		System.out.println( "fitSelectedVertices()" );
 		if ( pluginAppModel != null )
 		{
 			final MamutAppModel appModel = pluginAppModel.getAppModel();
 			final SourceAndConverter< ? > source = appModel.getSharedBdvData().getSources().get( sourceIndex );
 			if ( !( source.getSpimSource().getType() instanceof RealType ) )
 				throw new IllegalArgumentException( "Expected RealType image source" );
-
-			process( ( SourceAndConverter ) source );
-
-//			System.out.println( "fitSelectedVertices()" );
+			process( Cast.unchecked( source ) );
 		}
 	}
 
 	private static final boolean DEBUG = false;
 
```

Comment:
```suggestion
private static final boolean DEBUG_UI = false;
```
During testing with a large dataset, I figured out that it is handy to have a separate switch for debugging with/without UI.